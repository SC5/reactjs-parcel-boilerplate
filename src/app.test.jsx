import React from 'react';
import { shallow } from 'enzyme';
import App from './app';

let wrapper;

const defaultProps = {};

const getWrapper = (newProps) => {
  const props = { ...defaultProps, ...newProps };
  return shallow(<App {...props} />);
};

describe('<App />', () => {
  beforeEach(() => {
    wrapper = getWrapper();
  });

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
