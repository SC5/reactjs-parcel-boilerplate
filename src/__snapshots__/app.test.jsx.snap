// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
<div
  className="message-placeholder"
>
  <div
    className="message-placeholder__message"
  >
    Welcome to ReactJS World!
  </div>
</div>
`;
